source/common/json/config_schemas.cc

```diff
@@ -677,3 +675,285 @@ const std::string Json::Schema::ROUTER_HTTP_FILTER_SCHEMA(R"EOF(
     "additionalProperties" : false
   }
   )EOF");
+
+const std::string Json::Schema::CLUSTER_MANAGER_SCHEMA(R"EOF(
+  {
+    "$schema": "http://json-schema.org/schema#",
+    "definitions" : {
+      "sds" : {
+        "type" : "object",
+        "properties" : {
+          "cluster" : {"type" : "object"},
```

Comment:
So the check for cluster definition moved to ClusterFromJson. This is done to check the config of dynamically loaded clusters.